feature-libs/organization/administration/components/user/services/user-item.service.ts

```diff
@@ -19,11 +20,31 @@ import { CurrentUserService } from './current-user.service';
   providedIn: 'root',
 })
 export class UserItemService extends ItemService<B2BUser> {
+  // TODO: make b2BUserCreationNotifierService a required dependency
```

Comment:
Please create a Jira ticket for it, and but the number into the code comment.
https://sap.github.io/spartacus-docs/breaking-changes/#adding-new-constructor-dependencies-in-minor-versions

Reply:
Changed